workshops/splatter_paint/es-xl.md

```diff
@@ -0,0 +1,191 @@
+---
+name: 'Splatter Paint'
+description: 'Crazy colorful splatter paint in your browser with Paper.js'
+author: '@MatthewStanciu, @J-cordz'
```

Comment:
Your name shouldn't be here since you're not the author of this workshop.

Reply:
Hi, @sampoder mentioned that translator names should be on the translated author list.

Reply:
Already solved the typos and white spaces, thanks!

Reply:
Ohhh sorry!! I had no idea about that... then it's all good :-D